specificatie/adressering.yaml

```diff
@@ -98,3 +98,9 @@ components:
               type: boolean
             gebruikInLopendeTekst:
               type: boolean
+            datumIngangOnderzoekPersoon:
```

Comment:
dit is ook een allOf 'persoon.yaml#/components/schemas/InOnderzoek'. Daarin zit al datumIngangOnderzoek, die er dan denk ik uit kan. Moet je in deze dan niet het allOf verwijderen?

Reply:
done